sematic/tests/fixtures.py

```diff
@@ -24,14 +25,20 @@ def _set(key, value):
     def _get(key):
         return store[key]
 
+    def _set_from_file(key, file_path):
+        with open(file_path, "rb") as fp:
+            store[key] = b"".join(fp)
+
     storage.set = _set
     storage.get = _get
+    storage.set_from_file = _set_from_file
 
     try:
-        yield
+        yield store
```

Comment:
probably useful in some situations to have the store, so you can list keys easily